data/core/config.lua

```diff
@@ -2,15 +2,70 @@ local common = require "core.common"
 
 local config = {}
 
+---The frame rate of Lite XL.
+---Note that setting this value to the screen's refresh rate
+---does not eliminate screen tearing.
+---
+---Defaults to 60.
+---@type number
 config.fps = 60
+
+---Maximum number of log items that will be stored.
+---When the number of log items exceed this value, old items will be discarded.
+---
+---Defaults to 800.
+---@type number
 config.max_log_items = 800
+
+---The timeout, in seconds, before a message dissapears from StatusView.
+---
+---Defaults to 5.
+---@type number
 config.message_timeout = 5
+
+---The number of pixels scrolled per-step.
+---
+---Defaults to 50 * SCALE.
+---@type number
 config.mouse_wheel_scroll = 50 * SCALE
+
+---Enables/disables transitions when scrolling with the scrollbar.
+---When enabled, the scrollbar will have inertia and slowly move towards the cursor.
+---Otherwise, the scrollbar will immediately follow the cursor.
+---
+---Defaults to false.
+---@type boolean
 config.animate_drag_scroll = false
+
+---Enables/disables scrolling past the end of a document.
+---
+---Defaults to true.
+---@type boolean
 config.scroll_past_end = true
----@type "expanded" | "contracted" | false @Force the scrollbar status of the DocView
+
+---@alias config.scrollbartype
```

Comment:
I'd argue that it makes less sense because it applies to scrollbar in general. Even better, this should be documented in core.scrollview and borrow it here.

Reply:
Yeah, it is "defined" multiple times there, but never as an alias. I guess we should unify that then.